diffblue.yml

```diff
@@ -1,4 +1,2 @@
-branch: master
-language: java
-buildCmd: "mvn build"
-testCmd: "mvn cobertura:cobertura"
+buildCmd: "mvn test"
```

Comment:
This should be `mvn compile`.

`mvn test` will mean that the existing tests are built and then analysed, so we will generate tests to test tests, which is a bit of a mess.